admin-ui/app/utils/Util.js

```diff
@@ -14,3 +14,7 @@ export function getNewColor() {
   }
   return color
 }
+
+export function isEmptyObject(obj){
```

Comment:
I believe no need to create new function, we can use `isEmpty` from `lodash`
```JS
import isEmpty from 'lodash/isEmpty'
```

Reply:
Yeah that's right let me refactor that. Thanks!